development.md

```diff
@@ -0,0 +1,7 @@
+# Development
+
+The master branch of the handbook repo is published on the [Code For Nashville](https://github.com/code-for-nashville/code-for-nashville.github.io) website. The markdown is rendered into html using [Showdown.js](https://github.com/showdownjs/showdown).
```

Comment:
It seems like this would help if you are being really diligent and double checking that this page gets rendered correctly on the Code for Nashville site.  After the first time we render and confirm the handbook is rendered correctly on the Code for Nashville site, I don't think this will be helpful.  Am I missing another use case?  If not, I think we should skip committing this and the development server.